package.json

```
@@ -0,0 +1,25 @@
+{
+  "name": "rest-client-app",
+  "version": "0.1.0",
+  "private": true,
+  "scripts": {
+    "dev": "next dev --turbopack",
+    "build": "next build --turbopack",
```

Turbopack is still experimental: https://nextjs.org/docs/app/api-reference/turbopack#getting-started
It’s fine for dev, but for production builds we should stick to the stable command:
```suggestion
    "build": "next build",
```